JarrydWorland/2021_Capstone_GP2_Team
# Minimap

## Description
This PR adds a mini-map to aid the player in navigating the environment.

# Changes include
- Added mini-map (0275de5) 
- Added `Lerped` class (2a06a1f)
- Updated `CameraPanBehaviour` to use `Lerped` class (9c4cca4)

## How to test
Open the Gameplay scene and walk around, ensure that the mini-map correctly links rooms together and that the fog-of-war effect works as expected.
